transpiler/src/main/java/org/jsweet/transpiler/Java2TypeScriptTranslator.java

```diff
@@ -2178,7 +2178,7 @@ public void visitMethodDef(JCMethodDecl methodDecl) {
 			ClassType superClassType = (ClassType) parent.getExtendsClause().type;
 			MethodSymbol superMethod = Util.findMethodDeclarationInType(context.types, superClassType.tsym,
 					methodDecl.getName().toString(), (MethodType) methodDecl.type);
-			if (superMethod != null) {
+			if(superMethod != null && Util.isSourceElement(superMethod)) {
```

Comment:
Be sure to apply formatter before pushing please ;)

Reply:
I have seen it but I did not care...

Reply:
next time don't merge, I can fix, no problem